.github/workflows/update_xcm_data.yaml

```diff
@@ -0,0 +1,53 @@
+name: Update xcm coefficients
+
+on:
+  schedule:
+    - cron: '1 * * * *'
+  workflow_dispatch:
+
+
+jobs:
+  update-xcm:
+    runs-on: ubuntu-latest
+    outputs:
+      xcm_updated: ${{ steps.xcm.outputs.any_changed }}
+
+    steps:
+      - name: Checkout current repository to Master branch
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Checkout support repo
+        uses: actions/checkout@v2
+        with:
+          path: ./support
+          repository: nova-wallet/support-utils
+          ref: feature/auto-update-xcm
```

Comment:
Should we update it to master before merge?

Reply:
yes, you are right 👍